src/ui/repo-item-delegate.cpp

```diff
@@ -218,6 +222,27 @@ void RepoItemDelegate::paintRepoItem(QPainter *painter,
                       &repo_desc_rect);
     painter->restore();
 
+    // Paint repo sharing owner for private share
+    // TODO: show only for private share
+    if (item->repo().owner != seafApplet->accountManager()->accounts().front().username) {
```

Comment:
I know what, but `currentAccount` returns a new (and unnecessary) copy of Account while using `accounts().front()` doesn't (only reference)